install.sh

```diff
@@ -5,14 +5,14 @@ set -u
 # Check if script is run non-interactively (e.g. CI)
 # If it is run non-interactively we should not prompt for passwords.
 if [[ ! -t 0 || -n ${CI-} ]]; then
-  NONINTERACTIVE=1
+	NONINTERACTIVE=1
```

Comment:
Looks like most of these changes are spaces -> tabs conversion. Was this intentional or specific to your editor config?

Reply:
Unrelated to my editor. The changes are necessary for the `shell-fmt` [job](https://github.com/unionai/opta/blob/8fc65081c3a85091c941985a32e2ac8cbf6511e7/.github/workflows/ci.yml#L32) to pass on CI: https://github.com/unionai/opta/actions/runs/4387537338/jobs/7682951583

This is what the linter `shfmt` prefers, it looks like.